src/style.scss

```diff
@@ -0,0 +1,116 @@
+$primary: #11998e;
+$secondary: #38ef7d;
+$white: #fff;
+$gray: #9b9b9b;
+
+body {
+  background-image: linear-gradient(to right, #008770, #174ae6);
+}
+
+.gride {
+  display: flex;
+  flex-direction: row;
```

Comment:
- [ ] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

```
.d-flex {
   display: flex;
   flex-direction: row;
}
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉